### ISSUE TITLE
Linker warnings in Xcode 7 / iOS 9 SDK — built for newer iOS version (8.4) than being linked (8.0)

### ISSUE THREAD

**Issue by AdrianaPineda** [FredericJacobs/OpenSSL-Pod#20]
Hi. Im getting this warnings (a lot) when building my iOS app in Xcode 7 (simulator and devices).

```
ld: warning: object file (/path/to/lib.a(cryptlib.o)) was built for newer iOS version (8.4) than being linked (8.0)
```

**Comment by purrrminator** [comment]
+1

**Comment by AdrianaPineda** [comment]
Any updates on this?

**Comment by FredericJacobs** [comment]
You're using Xcode 7 so why aren't you linking against iOS 9?

**Comment by purrrminator** [comment]
@FredericJacobs do you mean I should change podfile platform to 9.0? I've tried this and got tons of new warnings from other pods: `CocoaLumberjack.a(DDASLLogCapture.o)) was built for newer iOS version (9.0) than being linked (8.0)`

**Comment by FredericJacobs** [comment]
These are warnings, will make sure to address for next version of OpenSSL (by setting a minimal SDK version). For now I wouldn't worry too much.

**Comment by purrrminator** [comment]
@FredericJacobs true, harmless warnings, but there are hundreds of them spamming the issue navigator.

**Comment by Vishal-0057** [comment]
@FredricJacobs : I am getting the same warning though My min SDK is iOS 7.0
object file (../Branch.framework/Branch(BNCEncodingUtils.o)) was built for newer iOS version (8.3) than being linked (7.0).

I have installed Branch SDK Without Pods.

**Comment by FredericJacobs** [comment]
Clear your Xcode cache and rebuild the project. It's not something related to OpenSSL.

**Comment by Vishal-0057** [comment]
I clean the code many times and built it but same warnings I am getting.

**Comment by FredericJacobs** [comment]
Not my problem ;)

**Comment by purrrminator** [comment]
The issue solved for me with 1.0.204.1. Thanks.
